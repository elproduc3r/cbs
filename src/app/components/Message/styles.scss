.message-container {
  margin-top: 18%;
  margin-left: 20%;
  max-width: 80%;
  color: white;
  font-size: 4rem;
  line-height: 4rem;
  font-weight: 800;
  text-transform: uppercase;
  display: flex;
  flex-direction: column;
  font-family: inherit;
  z-index: 3;
  position: relative;
  @media (max-width: 576px) {
    margin-left: 12%;
    font-size: 3rem;
    line-height: 3.25rem;
  }
}