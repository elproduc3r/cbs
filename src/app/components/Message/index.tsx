import React from 'react';
import StartButton from '../StartButton';
import "./styles.scss";

export interface MessageType {
  children: React.ReactNode;
};

const Message = (): JSX.Element => {
  return (
    <div className="message-container">
      Watch live
      <br />
      CBS Sports
      <StartButton />
    </div>
  );
};

export default Message;
