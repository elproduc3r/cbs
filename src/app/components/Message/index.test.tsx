import React from 'react';
import { render } from '@testing-library/react';
import Message from '.';

describe("<Message>", () => {
  test('snapshot', () => {
    const {container} = render(
      <Message />
    );
    expect(container).toMatchSnapshot();
  });  
});
