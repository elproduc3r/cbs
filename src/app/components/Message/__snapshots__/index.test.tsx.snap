// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Message> snapshot 1`] = `
<div>
  <div
    class="message-container"
  >
    Watch live
    <br />
    CBS Sports
    <button
      class="start-button"
    >
      Get Started
    </button>
  </div>
</div>
`;
